src/main/java/com/synopsys/integration/detect/configuration/DetectInfoUtility.java

```diff
@@ -46,7 +46,7 @@ public List<String> readDetectVersionFile() {
         }
     }
 
-    public String parseValueFromVersionFileContents(List<String> versionFileContents, String variableName) {
+    public static String parseValueFromVersionFileContents(List<String> versionFileContents, String variableName) {
```

Comment:
Does it need to be static since the invocation method is not static?

Reply:
I made this pre-existing method static just to conveniently call it without needing to instantiate a DetectInfoUtility object. The method simply fetches the active Detect version from a dynamically created text file

Reply:
This method was made static just for convenience sake and to avoid writing duplicate code. The version file itself is created + populated inside build.gradle and does not require instantiating DetectInfoutility class